static/styles/portico/markdown.css

```diff
@@ -86,6 +86,17 @@
         text-align: center;
     }
 
+    .key_symbol {
+        position: absolute;
+        left: 4%;
+        transform: translateX(-50%)
+    }
+
+    .key_text {
+        position: relative;
+        left: 5%;
+    }
```

Comment:
I might call these classes `legend_symbol` and `legend_label`.

Reply:
Thanks for the suggestion, will do.